### PR TITLE
Php + Ruby

### PR DESCRIPTION
**Pull request by scottrogowski**
Adding two languages at once

### CODE REVIEW

File: code2flow/model.py [hunk 1]
```diff
@@ -59,30 +64,24 @@ def _resolve_str_variable(variable, file_groups):
...
+
+    This function looks through all files to see if any particular node matches
+    the variable.points_to string
 
     :param Variable variable:
     :param list[Group] file_groups:
-    :rtype: Node|str
+    :rtype: Node|Group|str
     """
     has_known = False
```
Comment (suetsuguk) [inline code2flow/model.py:75]: `has_known` is never used and can be removed.

**Comment by suetsuguk** [comment]
https://github.com/scottrogowski/code2flow/blob/ddcae9b2b083d0381f4d8ecad75c766bc3e1759f/code2flow/python.py#L1

`as ast` is redundant and can be removed.

**Comment by suetsuguk** [comment]
https://github.com/scottrogowski/code2flow/blob/3150e087ebb3e004a50824aba2c8a9b6716d29bc/code2flow/engine.py#L575

'rb' and 'php' are missing in `choices`.

File: code2flow/php.py [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+        """
+        The "root_node" is an implict node of lines which are executed in the global
+        scope on the file itself and not otherwise part of any function.
+
+        :param lines list[ast]:
+        :param parent Group:
+        :rtype: Node
+        """
+        token = "(global)"
+        line_number = lines and lineno(lines[0]) or 0
```
Comment (suetsuguk) [inline code2flow/php.py:347]:
```suggestion
        line_number = lineno(lines[0]) if lines else 0
```

This seems to be a bit tricky. Can you use a ternary operator instead?

Comment (scottrogowski) [reply code2flow/php.py]: Yup. Ternary is a better choice.

**Comment by scottrogowski** [comment]
> https://github.com/scottrogowski/code2flow/blob/3150e087ebb3e004a50824aba2c8a9b6716d29bc/code2flow/engine.py#L575
> 
> 'rb' and 'php' are missing in `choices`.

@suetsuguk fixed

**Pull request merged**